src/REISE.jl

```diff
@@ -56,19 +64,20 @@ function run_scenario(;
         "storage" => storage,
         "interval_length" => interval,
         )
-    solver_kwargs = Dict("Method" => 2, "Crossover" => 0)
     # If a number of threads is specified, add to solver settings dict
     isnothing(threads) || (solver_kwargs["Threads"] = threads)
     println("All preparation complete!")
     # While redirecting stdout and stderr...
     println("Redirecting outputs, see stdout.log & stderr.err in outputfolder")
     redirect_stdout_stderr(stdout_filepath, stderr_filepath) do
         # Loop through intervals
-        interval_loop(env, model_kwargs, solver_kwargs, interval, n_interval,
-                      start_index, inputfolder, outputfolder)
+        interval_loop(optimizer_factory, model_kwargs, solver_kwargs, interval,
+                      n_interval, start_index, inputfolder, outputfolder)
         GC.gc()
-        Gurobi.finalize(env)
-        println("Connection closed successfully!")
+        if isa(optimizer_factory, Gurobi.Env)
+            Gurobi.finalize(optimizer_factory)
+            println("Connection closed successfully!")
```

Comment:
We don't need to do similar thing for other solvers or this is something to explore？

Reply:
I don't know that other solvers have a similar 'environment' feature that needs to be managed.